controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -221,6 +222,8 @@ func (r *SriovNetworkNodePolicyReconciler) syncDevicePluginConfigMap(ctx context
 		},
 		Data: configData,
 	}
+	controllerutil.SetControllerReference(dc, cm, r.Scheme)
```

Comment:
Are we ignoring the returned error on purpose? Not sure if it's worth to return the error, bit a log message would be useful.

Reply:
Will add a check as we do in other places for consitency.

technically there is no chance for errors to occur here.